notebooks/work-in-progress/explore_fuel_allocation.ipynb

```diff
@@ -2,8 +2,8 @@
  "cells": [
```

Comment:
<p>Legend to indicate what the colors mean. Axes labels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>